apps/web-giddh/src/app/shared/advance-receipt-adjustment/advance-receipt-adjustment.component.ts

```diff
@@ -112,6 +117,9 @@ export class AdvanceReceiptAdjustmentComponent implements OnInit {
             if (typeof this.invoiceFormDetails.voucherDetails.voucherDate !== 'string') {
                 this.invoiceFormDetails.voucherDetails.voucherDate = moment(this.invoiceFormDetails.voucherDetails.voucherDate).format(GIDDH_DATE_FORMAT);
             }
+            this.invoiceFormDetails.voucherDetails.tcsTotal = this.invoiceFormDetails.voucherDetails.tcsTotal ? this.invoiceFormDetails.voucherDetails.tcsTotal : 0
```

Comment:
You can use TypeScript default value expression, which will make the expression short to read. Same for below line 
```suggestion
            this.invoiceFormDetails.voucherDetails.tcsTotal = this.invoiceFormDetails.voucherDetails.tcsTotal || 0;
```